src/main/java/com/google/step/servlets/PromoCodeServlet.java

```diff
@@ -17,9 +18,13 @@
 
     private YouTubeInfoScraper infoScraper;
 
+    @VisibleForTesting
+    static final String REQUEST_PARAMETER = "formInput";
```

Comment:
nit: I guess static is not necessary here if you're accessing via an instance (`servlet.REQUEST_PARAMETER` vs. `PromoCodeServlet.REQUEST_PARAMETER`).